storage/database/db_migration.go

```diff
@@ -0,0 +1,122 @@
+// Modifications Copyright 2020 The klaytn Authors
+// Copyright 2015 The go-ethereum Authors
+// This file is part of go-ethereum.
+//
+// go-ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// go-ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with go-ethereum. If not, see <http://www.gnu.org/licenses/>.
+
+package database
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	dbMigrationFetchNum = 500
+)
+
+func (dbm *databaseManager) StartDBMigration(dstdbm DBManager) error {
+	// settings for quit signal from os
+	sigQuit := make(chan os.Signal, 1)
+	signal.Notify(sigQuit,
+		syscall.SIGHUP,
+		syscall.SIGINT,
+		syscall.SIGTERM,
+		syscall.SIGQUIT)
+
+	// TODO enable for all dbs
+	srcDB := dbm.dbs[0]
+	dstDB := dstdbm.getDatabase(DBEntryType(MiscDB))
+
+	// create src iterator and dst batch
+	srcIter := srcDB.NewIterator()
+	dstBatch := dstDB.NewBatch()
+
+	// vars for log
+	elapsedFetch, elapsedPut, elapsedTotal := time.Now(), time.Now(), time.Now()
+	var elapsedFetchTime time.Duration
+	iterateNum, fetchedTotal := 0, 0
+
+	// fetch keys and values
+	keys, vals, fetched := iterateDB(srcIter, dbMigrationFetchNum)
+	elapsedFetchTime = time.Since(elapsedFetch)
+	fetchedTotal += fetched
+
+loop:
+	for fetched > 0 {
+		// write fetched keys and values to DB
+		elapsedPut = time.Now()
```

Comment:
How about using `now` for `elapsedPut` and `elapsedFetch`?

Reply:
I've changed to `putStart` and `iterStart`. What do you think?

Reply:
@winnie-byun LGTM 😄